themes/default/views/admin/orders/show.blade.php

```diff
@@ -2,249 +2,282 @@
+            </h3>
+            <style>
+                th, td {
+                    padding: 5px;
+                }
+                .button {
+                    padding: 8px !important;
+                }
+            </style>
+            <table class="mt-4 gap-y-3 min-w-max overflow-x-auto">
+                <thead>
+                <tr class="border-b-2 border-secondary-200">
+                    <th class="font-bold">{{__('ID')}}</th>
+                    <th class="font-bold">{{__('Name')}}</th>
+                    <th class="font-bold">{{__('Cost')}}</th>
+                    <th class="font-bold">{{__('Status')}}</th>
+                    <th class="font-bold">{{__('Active to')}}</th>
+                    <th class="font-bold">{{__('Link')}}</th>
+                    <th class="font-bold text-end">{{__('Actions')}}</th>
+                </tr>
+                </thead>
+                <tbody>
                 @foreach ($order->products as $product)
-                    <div class="flex flex-row justify-between border-b">
-                        <div class="flex flex-col">
-                            <span class="font-bold">{{ $product->product()->get()->first()->name }}</span>
-                            <span class="text-sm">
-                                <button
-                                    class="text-blue-500 hover:text-blue-700 dark:text-blue-300 dark:hover:text-blue-500"
-                                    data-modal-show="changePriceQuantity{{ $product->id }}"
-                                    data-modal-target="changePriceQuantity{{ $product->id }}">
-                                    {{ __('Change Price/Quantity') }}
+                    <tr class="place-items-center">
+                        <td class="text-center">
+                            {{ $product->product()->get()->first()->id }}
+                        </td>
+                        <td class="text-center">
+                            {{ $product->product()->get()->first()->name }}
+                        </td>
+                        <td class="text-center">
+                            {{ $product->price ?? $product->product->price }} {{ config('settings::currency_sign') }}
+                        </td>
+                        <td class="text-center">
+                            @if($product->status === 'paid')
+                                <span class="text-green-500 font-bold">{{ __('Paid') }}</span>
+                            @elseif($product->status === 'pending')
+                                <span class="text-red-500 font-bold">{{ __('Pending') }}</span>
+                            @else
+                                <span class="text-yellow-500 font-bold">{{ ucfirst($product->status) }}</span>
+                            @endif
+                        </td>
+                        <td class="text-center">
+                            {{ $product->expiry_date ? $product->expiry_date : 'N/A' }}
```

Comment:
Not incorrect but you could use `??` instead when checking if a variable is not null.

For example:
`{{ $product->expiry_date ?? 'N/A' }}`

Reply:
Corrected to: {{ $product->link ?: 'N/A' }}, thanks